src/modules/rppi_validate.hpp

```diff
@@ -56,11 +56,9 @@ inline RppLayoutParams get_layout_params(RpptLayout layout, Rpp32u channels)
     }
     else if(layout == RpptLayout::NHWC || layout == RpptLayout::NDHWC)
     {
-        if (channels == 3) // PKD3
-        {
-            layoutParams.channelParam = 1;
```

Comment:
Why this change to remove the if (channels == 3)?

Reply:
@snehaa8 
Please add your observation here

Reply:
DeepCam had requirement for 16 channels thus had to remove (channels == 3) check